[Feature branch] Live graph doc (#3058)
grafana/alloy: docs/sources/troubleshoot/debug.md

### Graph page

{{< figure src="/media/docs/alloy/ui_graph_page_new_2.png" alt="Alloy UI graph page" >}}

The **Graph** page shows a graph view of components defined in the configuration file.
Clicking a component in the graph navigates to the [Component detail page](#component-detail-page) for that component.

> The components can be moved around by dragging them.

Reviewer: ```suggestion
You can click and drag the components to move them around.
```

Just trying to make the sentence more active and direct

> To access the graph page of a module, click on the Graph button on the module's detail page.

Reviewer: ```suggestion
To access the graph page of a module, click on the **Graph** button on the module's detail page.
```

> The amount of data that exits a component that supports [live debugging][#live-debugging-page] will be shown on the outgoing edges of the component.

Reviewer: ```suggestion
The amount of data that exits a component that supports [live debugging][#live-debugging-page] is shown on the outgoing edges of the component.
```

It's still passive using `is` here, but it's a little "better" than `will be`.
